lib/bootscale/file_storage.rb

```diff
@@ -13,11 +13,13 @@ def load(load_path)
 
     def dump(load_path, cache)
       path = cache_path(load_path)
-      return if File.exist?(path)
+      raise "This should not happen" if File.exist?(path)
```

Comment:
added test + made tests more reliable by running them in tmpdir instead of global /tmp